integration-tests/pom.xml

```diff
@@ -97,7 +97,32 @@
             <artifactId>guava</artifactId>
             <version>22.0</version>
         </dependency>
+        <dependency>
+            <groupId>io.github.bonigarcia</groupId>
+            <artifactId>webdrivermanager</artifactId>
+            <version>5.7.0</version>
+            <scope>test</scope>
+        </dependency>
 
+        <dependency>
```

Comment:
This block could use a comment explaining exactly which tests need this

Reply:
It's not for any particular test, it's because the WebDriverManager uses slf4j logging, so running any test will complain about missing implementation.

Reply:
Fair enough. Might still warrant a comment; you could ninja it in using the Github web interface

Reply:
Added the comment, needs a new approval.